### PR TITLE
Add container image

### PR DESCRIPTION
**Pull request by lcarva**
Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

**Comment by lcarva** [comment]
I built the image locally and pushed it to `quay.io/containerbuildsystem/operator-manifest:latest`. Once this PR is merged, I intend to setup builds on quay.io.

File: Dockerfile
Comment (MartinBasti) [inline Dockerfile:1]: `registry.access.redhat.com/ubi8/python-39:latest` Shouldn't be used this for unauthorized access? I cannot build image without auth.

Comment (MartinBasti) [reply Dockerfile:1]: Also to make Handolint happy and prevent breaking changes we should use more specific floating tag `python-39:1`

**Comment by MartinBasti** [comment]
Should be also following labels added?

`summary, description, io.k8s.description,  io.k8s.display-name, maintainer` and maybe a label pointing to this repo

File: Dockerfile
Comment (lcarva) [reply Dockerfile:1]: Done!

(Oddly enough, it wouldn't let me get this image from this registry before. Or maybe that was a slightly different image reference)

**Comment by lcarva** [comment]
> Should be also following labels added?
> 
> `summary, description, io.k8s.description, io.k8s.display-name, maintainer` and maybe a label pointing to this repo

I think I addressed this. Please, have another look.

**Review by ben-alkov — APPROVED** [review]
Nice :rocket:

**Review by mkosiarc — APPROVED** [review]

**Review by MartinBasti — APPROVED** [review]

**Pull request merged**
